### PR TITLE
Add perf tests for Random.cs.

### PR DESCRIPTION
**Pull request by stephenmichaelf**
Added missing performance tests for Random.cs.

This is part of the work to expand performance tests for System.Runtime.Extensions.

Issue: #18765

@karelz @JeremyKuhne @AlexGhiondea

### CODE REVIEW

File: src/System.Runtime.Extensions/tests/Performance/Perf.Random.cs [hunk 1]
```diff
@@ -49,5 +49,34 @@ public void Next_int_int()
...
+                    for (int i = 0; i < 40000; i++)
+                    {
+                        rand.NextDouble(); rand.NextDouble(); rand.NextDouble();
+                        rand.NextDouble(); rand.NextDouble(); rand.NextDouble();
+                        rand.NextDouble(); rand.NextDouble(); rand.NextDouble();
+                    }
+        }
     }
-}
+}
```
Comment (ViktorHofer) [inline src/System.Runtime.Extensions/tests/Performance/Perf.Random.cs]: No newline at end of file

Comment (stephenmichaelf) [inline src/System.Runtime.Extensions/tests/Performance/Perf.Random.cs]: Thanks! Added.

**Comment by stephenmichaelf** [comment]
Thanks for the review @joperezr! Could you merge? I don't have permission.

**Comment by safern** [comment]
@dotnet-bot Innerloop netfx Windows_NT Release x64 Build and Test (Hanged)
@dotnet-bot Innerloop netfx Windows_NT Debug x86 Build and Test

**Comment by safern** [comment]
@dotnet-bot test Innerloop netfx Windows_NT Release x64 Build and Test (Hanged)
@dotnet-bot test Innerloop netfx Windows_NT Debug x86 Build and Test

**Comment by danmosemsft** [comment]
@dotnet-bot test Innerloop netfx Windows_NT Release x64 Build and Test (Hanged)

**Pull request merged**